ad-hoc-timeline.html

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Ad-hoc Format Mentorship Timeline
+---
+<div class="timeline-page container">
+    <h1>Ad-hoc Relationship</h1>
+
+    <div class="container mt-6 mb-6">
+        <div class="row">
+            <div class="col-md-7 offset-md-3">
+                <ul class="timeline">
+                    <li>
+            
+                        <h4>Application period</h4>
+                        <div class="description">Every month, interested mentees will have the opportunity to apply to the one-time mentorship session program. The application process will be open for a specific period of time (a few weeks), and the applicants will need to provide information about their background, interests, and goals for the mentorship.</div>
+                    </li>
+                    <li>
+                        <h4>Mentee selection</h4>
+                        <div class="description">Once the application period has ended, the Mentorship Programme Team will review the applications and select the mentees who will participate in the program for the next month.</div>
+                    </li>
+                    <li>
+                        <h4>Mentor notification</h4>
+                        <div class="description">The Mentorship Programme Team will send a list of selected mentees to the mentors, along with their profiles and information about their goals and expectations for the mentorship.</div>
+                    </li>
+                    <li>
+                        <h4>Scheduling sessions</h4>
+                        <div class="description">Mentors will send a Calendly link and list of mentees to the Mentorship Programme Team, allowing these mentees to schedule sessions with the mentor directly.</div>
+                    </li>
+                    <li>
+                        <h4>Session preparation</h4>
+                        <div class="description">Before each session, mentors and mentees should prepare and plan for the meeting, setting clear goals and objectives for the session.</div>
+                    </li>
+                    <li>
+                        <h4>Session</h4>
+                        <div class="description">During the session, mentor and mentee should engage in meaningful and productive discussions, working towards the goals and objectives. Mentor should provide guidance, advice, and support to mentee, helping him/her to grow and develop his/her skills and knowledge.</div>
+                    </li>
+                    <li>
+                        <h4>Feedback</h4>
+                        <div class="description">After one-time session mentors will ask mentee to provide his/her feedback on the session and the program itself.</div>
```

Comment:
as a general question to @nora-weisser, should we really write `his/her` considering we only have women as mentees? 

another option is to write in a way we hides the gender, for example: 

```suggestion
                        <div class="description">After one-time session mentors will ask the mentee's feedback on the session and the program itself.</div>
```

Reply:
This is good suggestion! let's change it to hide gender!👍